.infrastructure/task-definitions/cms_app.json.tpl

```diff
@@ -10,8 +10,7 @@
 	"logConfiguration": {
         "logDriver": "awslogs",
         "options": {
-            "awslogs-create-group": "true",
-            "awslogs-group": "/ecs/cms-task-def",
+            "awslogs-group": "strapi-cms",
```

Comment:
I would provide it as variable
```suggestion
            "awslogs-group": "${log_group}",
```

Reply:
> I would provide it as variable

Ok, I must insert the var "cms_log_group" with value "strapi-cms" in the "variables.tf" file and "ecs.tf" file